mtgjson5/set_builder.py

```diff
@@ -404,7 +404,13 @@ def build_mtgjson_set(set_code: str) -> Optional[MtgjsonSetObject]:
     # Ensure we have a header for this set
     set_data = get_scryfall_set_data(set_code)
     if not set_data:
-        return None
+        with constants.RESOURCE_PATH.joinpath("additional_sets.json").open(
+            encoding="utf-8"
+        ) as f:
+            additional_sets_data = json.load(f)
+            set_data = additional_sets_data.get(set_code.upper(), False)
+        if not set_data:
```

Comment:
I don't see a need for this if statement tbh, as set_data is either set or it's None

Reply:
If set_data is None, then it needs to close the function and not try building the rest of the data with no inputs. Line 406 checks if the set code was valid in Scryfall, and line 412 checks if the set code is valid in the local file. If the set code is in neither repository, it returns None.